### PR TITLE
Feat/swarmion full stack example

### LINKED ISSUE #262: Relocate examples to Swarmion main repo

### PR DESCRIPTION
**Pull request by Sc0ra**
Fixes #262

### CODE REVIEW

File: examples/swarmion-full-stack/.all-contributorsrc [hunk 1]
```diff
@@ -0,0 +1,128 @@
+{
```
Comment (fargito) [inline examples/swarmion-full-stack/.all-contributorsrc:1]: You can remove this file

File: examples/swarmion-full-stack/LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Copyright 2021 François Farge, Adrien Cacciaguerra
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```
Comment (fargito) [inline examples/swarmion-full-stack/LICENSE.md:3]: You can remove this as well

File: examples/swarmion-full-stack/README.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
+# Swarmion example
```
Comment (adriencaccia) [inline examples/swarmion-full-stack/README.md:1]: Change this to a simpler one

File: examples/swarmion-full-stack/swarmion-starter.code-workspace [hunk 4]
```diff
@@ -0,0 +1,84 @@
+{
```
Comment (adriencaccia) [inline examples/swarmion-full-stack/swarmion-starter.code-workspace:1]: Change this filename to `examples/swarmion-full-stack/swarmion-full-stack.code-workspace`

**Review by adriencaccia — APPROVED** [review]

**Pull request merged**